elixir/apps/domain/lib/domain/component_versions/instance.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Domain.ComponentVersions.Instance do
```

Comment:
WDYT about naming it `Refresher` or `Fetcher`, instance doesn't say much about the purpose of the module:
```suggestion
defmodule Domain.ComponentVersions.Refresher do
```

Reply:
Sounds good to me.  I only chose `Instance` because I saw another module using that name and thought it might be a pattern you wanted to use.  This has been updated.

Reply:
And now I feel like I'm picky with you and doing 💩 myself, 🤣.